nlpsandboxclient/evaluation.py

```diff
@@ -42,16 +42,16 @@ def convert_annotations(self, annotations):
         new_annotations = {self.col: all_annotations}
         return new_annotations
 
-    def convert_dict(self, sys_file, gs_file):
-        with open(gs_file) as f:
-            gs = json.load(f)
-            gs = self.convert_annotations(gs)
-            gs = gs[self.col]
-        with open(sys_file) as f:
-            sys = json.load(f)
-            sys = self.convert_annotations(sys)
-            sys = sys[self.col]
-
+    def convert_dict(self, sys_file=None, gs_file=None, sys=None, gs=None):
+        if sys_file is not None and gs_file is not None:
+            with open(gs_file) as f:
+                gs = json.load(f)
+            with open(sys_file) as f:
+                sys = json.load(f)
```

Comment:
I think this makes the code more robust - since someone can "accidentally" input `sys_file=..., gs=`. Another thing here is that we are saying `sys_file` and `gs_file` always takes precedence over `sys` and `gs`.  So if someone specified all four parameters, only `sys_file` and `gs_file` is actually used.

```suggestion
        if gs_file is not None:
            with open(gs_file) as f:
                gs = json.load(f)
        if sys_file is not None:
            with open(sys_file) as f:
                sys = json.load(f)
```

Reply:
Thanks for pointing this out. It's a really good point!